docker/ubuntu/cpu_ispc_build/Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN apt-get -y update && apt-get install -y m4 bison flex zlib1g-dev libc6-dev-i
 # Configure ISPC build
 RUN mkdir build_$LLVM_VERSION
 WORKDIR build_$LLVM_VERSION
-RUN cmake ../ -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/usr/local/src/ispc/bin-$LLVM_VERSION
+RUN cmake ../ -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/usr/local/src/ispc/bin-$LLVM_VERSION -DCMAKE_CXX_FLAGS=-Werror
```

Comment:
Don't we need to modify other Dockerfiles as well?

Reply:
I don't think so. This Dockerfile is used in CI and that's why we need `-Werror` there. Though I don't have strong opinion here.